src/lib/Auth.ts

```diff
@@ -50,13 +52,32 @@ export const getApp = async () => {
     return { app };
 };
 
+function startTimer(){
+    Debug("Server timer starts")
+    Auth.timer_id = setTimeout(() => {
+        console.log(chalk.magenta('Server timeout: Please run fdk login command again.'));
+        Auth.stopSever()
+    }, SERVER_TIMER)
+}
+
+function resetTimer(){
+    if (Auth.timer_id) { 
+        Debug("Server timer reset")
+        clearTimeout(Auth.timer_id)
+        Auth.timer_id = null;
+    }
+}
 export const startServer = async () => {
     if (Auth.server) return Auth.server;
 
     const { app } = await getApp();
     const serverIn = require('http').createServer(app);
     Auth.server = serverIn.listen(port, (err) => {
-        if (err) console.log(err);
+        if (err) {
+            console.log(err);
```

Comment:
Log error in debug file too. @manishprajapati52

Reply:
Done